README.md

```diff
@@ -47,7 +47,7 @@ This version of the sample apps supports the latest Brightcove SDK and plugins.
 
 * [FreeWheel Widevine Modular Sample App](https://github.com/BrightcoveOS/android-player-samples/tree/master/brightcove-exoplayer/FreeWheelWidevineModularSampleApp): This app shows how to use the Brightcove ExoPlayer with FreeWheel ads and Widevine Modular content.
 
-Note that in order to enable the FreeWheel sample apps, you must independently obtain the non-free FreeWheel **AdManager.jar** library archive and install it into the directory **libs/** which is typically located at the root of your /home/ directory. The Brightcove Native SDK for Android and the FreeWheel sample apps require the FreeWheel AdManager library version 6.23.0. Please note that Brightcove does not distribute the FreeWheel libraries. Contact [FreeWheel](http://freewheel.tv/about/#contact-us) directly for more information on how to obtain their products.
+Note that in order to enable the FreeWheel sample apps, you must independently obtain the non-free FreeWheel **AdManager.aar** library archive and install it into the directory **libs/** which is typically located at the root of your /home/ directory. The Brightcove Native SDK for Android and the FreeWheel sample apps require the FreeWheel AdManager library version 6.28.0. Please note that Brightcove does not distribute the FreeWheel libraries. Contact [FreeWheel](http://freewheel.tv/about/#contact-us) directly for more information on how to obtain their products.
```

Comment:
Nit: @BCjwhisenant do you think we would need to match the exact library name here, like instead of `AdManager.aar` should be `AdManager-6.28.0.aar`?
or should it be the other way around and keep it general by renaming `AdManager-6.28.0.aar` to `AdManager.aar` inside the `~/libs/` folder and change all references?

Reply:
Not really, in my opinion. The AdManager archive is simply named "AdManager.aar." We've appended the version string for testing purposes, to distinguish it from other versions that may be hanging around in the ~/libs folder.

Reply:
I just want to be clear on what's expected to build the FreeWheel sample apps.
For example, for someone new, adding the AdManager.aar file to the ~/libs dir won't enable the FreeWheel samples apps. They will need to dive into the gradle settings to figure out that they must renamed the library to AdManager-6.28.0.aar in order for the samples app to be built.

People from support have already reached out asking why the FreeWheel sample apps are not enabled in AS by default.

Reply:
addressed below.